src/javascript/main.js

```diff
@@ -27,29 +27,51 @@ __webpack_public_path__ =
     window.contextJsParameters.contextPath +
     '/modules/dx-commons-webpack/javascript/apps/';
 
-function jsload(path) {
-    return new Promise(resolve => {
-        console.debug(`Load ${path}`);
-        var scriptTag = document.createElement('script');
-        scriptTag.src = window.contextJsParameters.contextPath + path;
-        scriptTag.onload = function () {
-            resolve(path);
-        };
+const jsload = path => new Promise(resolve => {
+    console.debug(`Load ${path}`);
+    let scriptTag = document.createElement('script');
```

Comment:
Do you think we can do async import here now ?

It will simplify a lot this code.

Reply:
don't understand this , any example ?

Reply:
`const jsLoad = import;` 

To sum up :p

Reply:
something like `import 'http://modules/jahia-administration/javascript/apps/jahia.bundle.js'` ? directly in the browser ? is it supposed to be supported anywhere ?
Btw it would be catched and interpreted by webpack. Webpack itself uses <script> tag injection and it seems to be the most common way to do that for now ?

Reply:
Ok, clearly not possible `Uncaught SyntaxError: Cannot use import statement outside a module` :-)

Reply:
I have a `Uncaught (in promise) ReferenceError: dx_commons_export is not defined
` when using import in place of script injection. Not sure it's an easy replacement